sdks/python/test-suites/gradle.properties

```diff
@@ -21,28 +21,30 @@
 # And if you want to use multiple versions, concat them with comma and no whitespace.
 
 # dataflow test-suites
-dataflow_precommit_it_task_py_versions=3.7,3.9
+# (TODO): https://github.com/apache/beam/issues/21971
+# Add python 3.10 to dataflow test-suites
+dataflow_precommit_it_task_py_versions=3.7,3.9,3.10
 dataflow_mongodbio_it_task_py_versions=3.7
 dataflow_chicago_taxi_example_task_py_versions=3.7
-dataflow_validates_runner_batch_tests=3.7,3.9
-dataflow_validates_runner_streaming_tests=3.7,3.8,3.9
-dataflow_validates_container_tests=3.7,3.8,3.9
+dataflow_validates_runner_batch_tests=3.7,3.9,3.10
```

Comment:
The `dataflow_validates_runner_batch_tests` and `dataflow_validates_runner_streaming_tests` are intended for Dataflow Runner v1. But since most of the Python jobs default to runner_v2, these also run on runner_v2. So I add an explicit parameter `disable_runner_v2` for these tests and updated the python versions to be `3.7` and `3.9` for these tests